index.html

```diff
@@ -0,0 +1,499 @@
+            <div class="feature-name color-black">name</div>
+          </div>
+          <p class="feature-para color-black para1">para</p>
+          <p class="feature-para color-black para2">para</p>
+          <p class="feature-para color-black para3">para</p>
+          <span class="feature-span color-black"></span>
+          <div class="feature-tag flex-row">
+            <div class="tag color-black"></div>
+            <div class="tag color-black"></div>
+            <div class="tag color-black"></div>
+          </div>
+          <i class="fas fa-long-arrow-alt-right fa-3x bottom"></i>
+        </div>
+        <div class="featured-btm-right">
+          <div class="feature-writer flex-row">
+            <div class="feature-img rotate25 clearrotate color-black">image</div>
+            <div class="feature-name color-black">name</div>
+          </div>
+          <p class="feature-para color-black para1">para</p>
+          <p class="feature-para color-black para2">para</p>
+          <span class="feature-span"></span>
+          <div class="feature-tag flex-row">
+            <div class="tag color-black"></div>
+            <div class="tag color-black"></div>
+          </div>
+          <i class="fas fa-long-arrow-alt-right fa-3x bottom"></i>
+        </div>
+      </div>
+    </div>
+
+    <!-- section for the first round of post -->
+
+    <section>
+      <div class="posts-container flex-row-jcc">
+        <div class="box article-color">
+          1
+          <div class="time"></div>
+          <h1 class="title">Lorem ipsum</h1>
+          <h2 class="author">Lorem ipsum</h2>
+          <article>
+            <p class="content"></p>
+          </article>
+          <i class="fas fa-long-arrow-alt-right fa-3x bottom"></i>
+        </div>
+        <div class="box article-color">
+          2
+          <div class="time"></div>
+          <h1 class="title">Lorem ipsum</h1>
+          <h2 class="author">Lorem ipsum</h2>
+          <article>
+            <p class="content"></p>
+          </article>
+          <i class="fas fa-long-arrow-alt-right fa-3x bottom"></i>
+        </div>
+        <div class="box article-color">
+          3
+          <div class="time"></div>
+          <h1 class="title">Lorem ipsum</h1>
+          <h2 class="author">Lorem ipsum</h2>
+          <article>
+            <p class="content"></p>
+          </article>
+          <i class="fas fa-long-arrow-alt-right fa-3x bottom"></i>
+        </div>
+      </div>
+
+      <!-- middle section -->
+      <div class="middle-comntainer flex-row fourth-color">
+        <div class="middle-images article-color">
+          <div class="signin"></div>
+          <div class="shopify-container flex-col">
+            <div class="shopiitems flex-col-aic">
+              <p class="person-para color-black"></p>
+              <p class="person-para-last color-black"></p>
+            </div>
+            <!-- <div class="shopiitems"></div> -->
```

Comment:
- [ ] It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))